packages/lib/ren-tx/src/configs/genericMint.ts

```diff
@@ -395,11 +464,12 @@ export const mintConfig: Partial<MachineOptions<GatewayMachineContext, any>> = {
                 delete (machineContext as any).depositMachines;
                 machines[evt.data.sourceTxHash] = spawnDepositMachine(
                     machineContext,
-                    `${String(evt.data.sourceTxHash)}DepositMachine`,
+                    `${String(evt.data.sourceTxHash)}`,
```

Comment:
This can be simplified now that there's no concatenation.

Reply:
Ah, good catch, will do